packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -337,4 +344,20 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
       this.allOptions?.length ? '' : 'slds-hidden'
     }`;
   }
+
+  formatRecentQuery(recentQuery, query) {
```

Comment:
`formatRecentQuery` is now defined in this component directly instead of the Quantic Utils to avoid a circular dependency now that `formatRecentQuery` needs to import the function `getHeadlessBundle` from the QuanticHeadlessLoader.

if we keep  formatRecentQuery in QuanticUtils we have:
QuanticUtils imports from QuanticHeadlessLoader and QuanticHeadlessLoader imports from QuanticUtils. 

to avoid breaking changes, I marked `formatRecentQuery` in the Quantic Utils as `deprecated` and I will create the ticket to remove it completely in the v4 once this PR is approved.

Reply:
those are valid points! I think it would be a good idea to cover this method with unit testing